### ISSUE TITLE
In hardcore mode, Attrition should increase over time instead of decreasing.

### ISSUE THREAD

**Issue by James103** [stormdirus2/Memento-Mori#1]
Currently, when you die in Hardcore mode with Requiem installed, you respawn as a spirit once and gain some Attrition, which fades like normal. I propose a solution to make Attrition actually play a bigger role post-death in Hardcore.

My proposal is basically this:
* When you die in Hardcore mode, you gain a level of Attrition immediately.
* Whenever your Attrition runs out in Hardcore mode, it **increases a level** instead of decreasing, while resetting the timer.
* Once you reach Attrition V, the mob you're possessing dies and you can no longer possess anything.
* One level later, at Attrition VI, you (the spirit) instantly dies with the standard Requiem hardcore death.

In other words, in Hardcore mode, not only do you respawn as a spirit once (then die for real), but also your Attrition increases over time, until your soul is completely faded (at level VI) and you die for real.
